trestle/cli.py

```diff
@@ -12,14 +12,41 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Test cli to print a simple test message."""
+"""Starting point for the Trestle CLI."""
 
-import logging
+from ilcli import Command
+
+from trestle.__init__ import __version__
+from trestle.core.commands.add import AddCmd
+from trestle.core.commands.assemble import AssembleCmd
+from trestle.core.commands.create import CreateCmd
+from trestle.core.commands.import_ import ImportCmd
+from trestle.core.commands.init import InitCmd
+from trestle.core.commands.merge import MergeCmd
+from trestle.core.commands.replicate import ReplicateCmd
+from trestle.core.commands.split import SplitCmd
+from trestle.core.commands.validate import ValidateCmd
+
+
+class Trestle(Command):
+    """Manage OSCAL files in a human friendly manner."""
+
+    subcommands = [InitCmd, CreateCmd, SplitCmd, MergeCmd, ReplicateCmd, AddCmd, ValidateCmd, ImportCmd, AssembleCmd]
```

Comment:
See the comment I had above w.r.t a single module. I think we could also do a recursive import rather than having to maintain a list (but that's just been tricky and does not need to be solved now).